sections/featured-blog.liquid

```diff
@@ -51,14 +51,42 @@
         {%- endif -%}
       </div>
     {%- endunless -%}
-    {%- if section.settings.blog != blank and section.settings.blog.articles_count > 0 -%}
+
       <slider-component class="slider-mobile-gutter{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
```

Comment:
I _think_ this whole node can use one less indent. If you format the file with Prettier it should handle this for you. We should make sure that's setup for you.

Reply:
I just reloaded my prettier and it re-formated my code, thank you :).